src/pretix/base/payment.py

```diff
@@ -190,6 +198,13 @@ def payment_form(self, request: HttpRequest) -> Form:
         form.fields = self.payment_form_fields
         return form
 
+    def _is_still_available(self):
+        availability_date = self.settings.get('_availability_date', as_type=datetime)
+        if availability_date is not None and datetime.now() > availability_date:
```

Comment:
Please use `from django.utils.timezone import now`, and `now() > availability_date` which uses sensible timezone settings. :)

Also, `now` is a datetime object while the availability date is a date object, is that intended?